flask_api/tests/test_mediatypes.py

```diff
@@ -6,7 +6,7 @@
 
 class MediaTypeParsingTests(unittest.TestCase):
     def test_media_type_with_params(self):
-        media = MediaType('application/xml; schema=foobar, q=0.5')
+        media = MediaType('application/xml; schema=foobar; q=0.5')
```

Comment:
Probably correct, mind linking me to an RFC or similar to refresh myself on this?